### PR TITLE
ceph-grafana-trigger: update the grafana version

### PR DESCRIPTION
**Pull request by nizamial09**
after https://github.com/ceph/ceph/pull/50845/

### CODE REVIEW

File: ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@
       - string:
           name: CONTAINER_VERSION
           description: "The version tag for the containers; will have -ARCH added to it.  By convention, the version of Grafana"
-          default: "8.3.5"
+          default: "9.4.7"
```
Comment (epuertat) [inline ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml:26]: That's quite a leap! Have we checked that the Dashboards look ok with this version? I checked the [9.0 release notes](https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v9-0/) and don't see any breaking changes, but it won't do any harm to check out.

**Review by epuertat — APPROVED** [review]
LGTM

File: ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml (hunk 1 above)
Comment (nizamial09) [reply ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml:26]: yup, i locally built the image and tested it with ceph-dev.

Comment (nizamial09) [reply ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml:26]: here is the [image I built](https://quay.io/repository/nizamial09/ceph-grafana?tab=tags).

**Pull request merged**